### DISCUSSION TITLE
Highlight area when clicking or hover over text

### DISCUSSION THREAD

**Discussion by srednasoft** [img-mapper/img-mapper#51] [Q&A] [answered]
Hi, 

I have a floor map with rooms that can be highlighted and all the rooms are also listed with names at the side of the image, like a menu. If I click or hover over a roomname in the list I would like the room to highlight in the image and also show a tooltip with room info.

I'll guess the tooltip will be another plugin, but can I highlight areas via other objects like described over?

Thnx
Anders

**Comment by NishargShah** [comment, marked as answer]
Yes, you can highlight it using `preFillColor` property.

Codesandbox: https://codesandbox.io/s/vigilant-pascal-5xzrro
